test/DryIoc.Issue139_ReducedLoadTests/ReducedStackoverflow.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net471</TargetFramework>
+        <TargetFramework>net472</TargetFramework>
```

Comment:
I had to increase this to .Net  4.7.2 because .Net installer did not allow me to install .Net 4.7.1 when I had 4.7.2 installed on my computer. And visual studio was failing to compile... :/